### DISCUSSION TITLE
Show character count instead of word count

### DISCUSSION THREAD

**Discussion by derWalter** [Zettlr/Zettlr#4272] [Q&A] [answered]
Hey Guys,

maybe I've just missed this one so far, but at my university everything is in character counts
and not word counts anymore. So because of this I would like to set my goals in characters,
and also show characters (inlcuding spaces) instead of words on the top right ( I am aware though,
that I can click on the word count and then see the character count).

Does that work (like setting the units in imperial/inches and metric/cm in software :P) and I missed it,
or is this feature not available.

thanks for your time!

Walter

**Comment by nathanlesage** [comment, marked as answer]
There's a setting "count characters instead of words" in the settings that does precisely that.

**Comment by derWalter** [comment]
guess the "(e.g. for chinese)" made me skip it instead of realizing it :D
